src/core_modules/capture-core/components/Pages/common/WidgetEventEditWrapper/WidgetEventEditWrapper.js

```diff
@@ -5,13 +5,23 @@ import { pageStatuses } from '../../EnrollmentEditEvent/EnrollmentEditEventPage.
 import { IncompleteSelectionsMessage } from '../../../IncompleteSelectionsMessage';
 import { WidgetEventEdit } from '../../../WidgetEventEdit';
 import type { Props } from '../../../WidgetEventEdit/widgetEventEdit.types';
+import {
+    useMetadataForEnrollmentEventForm,
+} from '../../../WidgetEnrollmentEventNew/DataEntry/hooks/useMetadataForEnrollmentEventForm';
```

Comment:
`useMetadataForEnrollmentEventForm` and `useMetadataForSingleEventForm` seem like very similar files. Could we use only one of them? And put it in `DataEntries/common/ProgramStage` maybe?

Reply:
That was the original plan, but I thought the code was a bit difficult to read. I have refactored them both into one and moved it to `DataEntries/common/ProgramStage`. I have made the assumption that if you do not pass in a stageId, we assume it's an event program, but if you pass in a stageId, we assume it's a tracker program stage. Have a look and see if it makes sense.